packages/local/src/index.ts

```diff
@@ -0,0 +1,161 @@
+import { Context, Logger, Random, Schema } from 'koishi'
+import { ImageSource } from 'koishi-plugin-booru'
+import { LocalStorage } from './types'
+import { createHash } from 'node:crypto'
+import { PathOrFileDescriptor, existsSync, readFileSync, readdirSync, statSync, writeFile } from 'node:fs'
+import { basename, extname, isAbsolute, join, resolve, sep } from 'node:path'
+import { pathToFileURL } from 'node:url'
+
+class LocalImageSource extends ImageSource<LocalImageSource.Config> {
+  languages = ['en', 'zh-CN', 'ja']
```

Comment:
Consider making this into config because the languages here would be bounded by users theirselves.